src/dss/main.py

```diff
@@ -31,6 +33,47 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+IMAGE_OPTION_HELP = "\b\nThe image used for the notebook server.\n" + RECOMMENDED_IMAGES_MESSAGE
+
+
+@main.command(name="create")
+@click.argument(
+    "name",
+    required=True,
+)
+@click.option(
+    "--image",
+    default=DEFAULT_NOTEBOOK_IMAGE,
+    help=IMAGE_OPTION_HELP,
+)
+# FIXME: Remove the kubeconfig param from the create command (and any tests) after
+#  https://github.com/canonical/data-science-stack/issues/37
+@click.option(
+    "--kubeconfig",
+    help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
+)
+def create_notebook_command(name: str, image: str, kubeconfig: str) -> None:
+    """Create a Jupyter notebook in DSS and connect it to MLflow.
+    This command also outputs the URL to access the notebook on success.
+
+    \b
+    Examples:
+        dss create my-notebook --image=pytorch
+        dss create my-notebook --image=kubeflownotebookswg/jupyter-scipy:v1.8.0
```

Comment:
Help message should also provide list of recommended images. Check the ux [spec](https://docs.google.com/document/d/1v0wU8TBb-ui6RcjGRbLwA6XFZnWbuiVb2TPmEgwKejY/edit#heading=h.wylzoqh1ph5w) for refference.